### PR TITLE
fix package.json scripts to allow yarn installation directly from github

### PR DESCRIPTION
**Pull request by ohler**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "test": "jest",
     "tsm": "ts-node ./lib/cli.ts",
     "clean": "rm -rf ./dist",
-    "build": "yarn clean && tsc",
```
Comment (skovy) [inline package.json:25]: why is this necessary?

File: package.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
...
-    "prepack": "yarn build",
+    "build": "yarn clean && tsc && chmod +x dist/lib/cli.js",
```
Comment (skovy) [inline package.json:25]: why is this necessary?

Comment (ohler) [reply package.json:25]: on my system, when installing from github, the `tsm` binary installed by yarn is just a symlink to `dist/lib/cli.js`:
```
$ ls -la node_modules/.bin/tsm 
lrwxr-xr-x  1 ohler  staff  37 Jun 29 12:58 node_modules/.bin/tsm -> ../typed-scss-modules/dist/lib/cli.js
```

without the `chmod +x`, the `tsm` command doesn't work.
